fetch-in-react/src/app.jsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-unused-vars -- Remove this line. */
+import React, { useState, useEffect } from 'react';
+import UserList from './user-list';
+
+export default function App() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [users, setUsers] = useState([]);
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+    fetch('https://jsonplaceholder.typicode.com/users')
+      .then((response) => response.json())
+      .then((data) => {
+        setUsers(data);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.error('There was an error!', error);
+      });
```

Comment:
If the network data fails to load, make sure your user can see it.

Reply:
![image](https://user-images.githubusercontent.com/117887912/218585913-85b96df3-bd10-408f-99c3-c94b6c5fcabd.png)